gguf-py/gguf/constants.py

```diff
@@ -786,7 +789,7 @@ def get_type(val: Any) -> GGUFValueType:
             return GGUFValueType.INT32
         # TODO: need help with 64-bit types in Python
         else:
-            print("Unknown type:", type(val))
+            logger.error(f"Unknown type: {type(val)}")
             sys.exit()
```

Comment:
This module does not need a logger. This should really be:
```suggestion
            raise ValueError(f"Unknown type: {type(val)}")
```